Classes/Controller/CartController.php

```diff
@@ -522,10 +522,10 @@ public function initializeOrderCartAction()
         ) {
             $fields = $this->pluginSettings['validation']['orderCartAction']['fields'];
 
-            if (key_exists('acceptTerms', $fields)) {
+            if (array_key_exists('acceptTerms', $fields)) {
                 $this->setDynamicValidation('acceptTerms', $fields['acceptTerms']);
             }
-            if (key_exists('acceptConditions', $fields)) {
+            if (array_key_exists('acceptConditions', $fields)) {
```

Comment:
yes, but is shown as deprecated.
but the main reason is: you are already using array_key_exists on other places in code